site/content/entry/yuzu-progress-report-feb-2021/index.md

```diff
@@ -0,0 +1,223 @@
+This stems from the fact that resources are finite, especially in weaker hardware such as that in the Nintendo Switch.
+A PC, on the other hand, isn't as restricted as a Nintendo Switch.
+Until now, whenever a process requested resources, yuzu would create its own instance of `resource limit` instead of using a system-wide variable to keep track of it.
+This PR is just the initial step in preparation to reverse engineer the correct behaviour and implement it in the emulator.
+
+## Paint me like one of your french bits
+
+[epicboy](https://github.com/ameerj) has also been busy this month implementing two features through [compute kernels](https://en.wikipedia.org/wiki/Compute_kernel), a special kind of program that is written to run in the GPU instead of the CPU.
+Originally, these subroutines were used to calculate light levels, darkness, colours, and other properties to render 3D images on the screen.
+Thus, these programs were promptly named as [shaders](https://en.wikipedia.org/wiki/Shader).
+
+Modern GPUs are designed to break down their workload into smaller sized problems, which in turn are processed simultaneously in the many compute units of the card (entities akin to cores in a CPU).
+The reason for this design choice is simply because parallelisation is a very efficient scheme to process computer graphics. Since a single instruction is capable of operating over many components of data at the same time, such as the vertices and textures of a 3D scene, this increases the throughput of information, especially when compared against the performance of running the same operations on a CPU.
+Their potential isn't limited to just these functions though.
+It is possible to write programs that won't necessarily operate over graphics, yet still benefit from these characteristics.
+This is known as `GPGPU` - [General-purpose computing on graphics processing units](https://en.wikipedia.org/wiki/General-purpose_computing_on_graphics_processing_units) - and it's intended to be used when there is a problem that can be separated into a number of parallel tasks in order to be processed more efficiently. These problems are commonly called `embarrassingly parallel problems`.
+
+One of these cases was the [use of compute shaders to decode ASTC textures](https://github.com/yuzu-emu/yuzu/pull/5927).
+`ASTC` stands for "Adaptable Scalable Texture Compression," and it's a fairly new image compression format mainly aimed at mobile devices.
+The Nintendo Switch is capable of decoding these textures natively in hardware, but it's a feature that most PC GPU vendors lack in their products (with the exception of Intel Graphics, being the only vendor that offers native support).
+The decoding of these textures is therefore a non-trivial task that can have a huge impact on performance, as seen in games such as `Astral Chain` and `Luigi's Mansion 3`.
```

Comment:
What about

> The decoding of these textures is therefore a non-trivial task that can have a huge impact on performance. Two notable examples are `Astral Chain` and `Luigi's Mansion 3`, since both games make extensive use of this format, but it can also be observed to various degrees in other titles, where these textures are generally used to a lesser degree in menu icons, minimaps, etc.

Reply:
perfect!